book.rb

```diff
@@ -0,0 +1,11 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rental
+
+  def initialize(title, author)
+    super()
+    @title = title
+    @author = author
+    @rental = []
+  end
+end
```

Comment:
- [ ] There should be a method in your book class ` add_rental` which takes `person` and `date` as arguments, this method should add the book to the rental class